contextproject-ms3/src/main/java/contextproject/App.java

```diff
@@ -54,12 +58,16 @@ public void start(Stage stage) throws Exception {
     stage.setTitle("Cool demo!");
     stage.setScene(scene);
     stage.show();
+    
+    LibraryLoader libraryLoader = new LibraryLoader("library.xml");
+    libraryLoader.load();
```

Comment:
If no library file is present, this should create a new library instead of returning null. Basically:
```
    try {
      library = libraryLoader.load();
    } catch (FileNotFoundException e) {
      library = new Library();
    }
```